pkg/manifest/os.go

```diff
@@ -22,6 +24,7 @@ import (
 	"github.com/osbuild/images/pkg/platform"
 	"github.com/osbuild/images/pkg/rhsm/facts"
 	"github.com/osbuild/images/pkg/rpmmd"
+	"github.com/sirupsen/logrus"
```

Comment:
(nipick) in general our convention for imports is (in most go project btw):
- stdlib
- external third party
- our own includes

we are not very strict about it but here I would suggest to move it between "strings" and github.com/osbuild/images/internal/common" (with \n before/after)